python/starter_kit/en/START_WELL.md

```diff
@@ -0,0 +1,67 @@
+(for some reasons hard to explain here, but for instance, the language does not enforce the usage of `;` after each expression).
+
+
+### Some good resources to learn well
+
+> On the Internet, lots of tutorials/courses to learn development exist, except many
+> of them are bad because they teach you bad practices or 
+> obsolete things. - [learndev.info](https://www.learndev.info/en)
+
+Indeed, the resources available on the Internet are not always relevant or up-to-date.
+That is why, the website [learndev.info](https://www.learndev.info/en) offers adapted and relevant courses,
+such as the course of **[ TODO ]**.
+
+
+
+### Errors to not do when you begin
+
+Many people begin programming with the idea that is not necessary complicated,
+and they can quickly make very big projects without the need to aquire all the bases.
+Well, that is mostly a big mistake.
+
+In fact, learn a language whether it is Python or another, needs to be done carefully.
+You should not try to drive a car without learn to do it. Thus, a lot of beginners 
+in Python launch themselves in libraries too much advanced for them, like `Pygame`, `Tkinter` or such as `discord.py`
+that need advanced notions such as the Oriented Object Programming, the error handling,
+decorators or even the manipulation of threads/async.
+
+If you do not know what is a function, an object or manipulate loops likes `for` and `while` loops,
+you have everything to gain to let your project on a side for a time to consolidate your knowledge and even, make
+projects at your scale in order to accumulate knowledge for the one that is close to your heart.
+Do not be afraid to let a project during several months.
+Accumulate frustration by continuously failling on a too ambicious project can disgust you with
+programming in general (that would be counterproductive).
+
+
+### Projects, lots of projects, more projects!
+
+What is better to begin with programming more than give yourself objectives. Have a project
+is a very great way to learn. Indeed, after seeing the theory in courses,
+begin a project, permits to practice. Practicing is the only way that permits to remember durably.
+Thus, we can begin small and adapted projects to what we learned.
+
+Here are some examples:
+- A fair price game after seeing loops and conditions ;
+- A hangman, when you master the treatment of character strings ;
+- A tic-tac-toe in shell, pretty fun and instructive to assimilate the loops.
+
+Once the basis mastered, do not hesitate to be curious. Relying on your knowledge does not permit to evolve.
+There are always things to learn, to experiment or to discover.
+
+### Conclusion
+
+Python is a really good language to begin in the programming world. Moreover to be easy to learn,
+it permits a huge amount of things. Dispite the fact, it is necessary to practice with the help of projects,
+it is very discouraged to begin with something too big for yourself.
```

Comment:
```suggestion
it is very discouraging to begin with something too big for yourself.
```

Reply:
In fact, the French version is "Il est déconseillé", your change is for "Il est décourageant".
Both versions are true, but it may be better to use "it is not advisable to" for a better translation.